README.md

```diff
@@ -0,0 +1,16 @@
+# Cloud Endpoints DNS Module for Terraform
```

Comment:
It would be nice to have the README explain a bit more about the use case (or, if you are writing a solution for this, link to it).

Reply:
Done.